pkg/js/parse_tests/014-caa.json

```diff
@@ -0,0 +1,48 @@
+{
+    "registrars": [],
+    "dns_providers": [],
+    "domains": [
+        {
+            "name": "foo.com",
+            "registrar": "none",
+            "dnsProviders": {},
+            "records": [
+                {
+                    "type": "CAA",
+                    "name": "@",
+                    "caatag": "issue",
+                    "caaflag": 0,
+                    "target": "letsencrypt.org"
+                },
+                {
+                    "type": "CAA",
+                    "name": "@",
+                    "caatag": "issuewild",
+                    "caaflag": 0,

```

Comment:
Ah, that's a typo. The field is actually `caatags`. (I didn't really test this file before I push as master already have it broken)